charts/prometheus-pushgateway/Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v1
 appVersion: "1.2.0"
 description: A Helm chart for prometheus pushgateway
 name: prometheus-pushgateway
-version: 1.4.2
+version: 1.4.3
```

Comment:
New features should raise minorv version, not bugfix -> 1.5.0

Reply:
Ok thanks for this!